CONTRIBUTING.rst

```diff
@@ -2,8 +2,8 @@
 Contributing
 ============
 
-.. _prs: https://github.com/dr-prodigy/python-holidays/pulls
-.. _`beta branch`: https://github.com/dr-prodigy/python-holidays/tree/beta
+.. _prs: https://github.com/vacanza/python-holidays/pulls
+.. _`beta branch`: https://github.com/vacanza/python-holidays/tree/beta
 .. |contributors| image:: https://img.shields.io/github/contributors/dr-prodigy/python-holidays
     :target: https://www.github.com/dr-prodigy/python-holidays
```

Comment:
Why doesn't this link to https://github.com/vacanza/python-holidays/graphs/contributors ?

Reply:
Included 👍